packages/repository-modernizer/src/create-base-project-structure.js

```diff
@@ -182,6 +182,22 @@ var CreateBaseProjectStructure = {
                     `CreateBaseProjectStructure: Base parent pom.xml created at ${projectPath}`
                 );
             }
+            if (projects.length > 1) {
+                fs.copyFileSync(
+                    path.join(basePath, constants.BASE_PARENT_POM),
+                    path.join(projectPath, constants.POM_XML)
+                );
+                conversionStep.addOperation(
+                    new ConversionOperation(
+                        commons_constants.ACTION_ADDED,
+                        path.join(projectPath, constants.POM_XML),
+                        "Created project's base parent pom.xml"
```

Comment:
I would call it a `base reactor pom.xml`.

Reply:
Hi @ManasMaji, Thanks for reviewing, all the feedbacks are incorporated.
Also, as long as the project's parent pom.xml remains unique (not multiple), the changes will work eg. in the case of National Instrument, the parent pom is https://git.corp.adobe.com/acsskylinepilots/national-instruments/blob/dev/content-aem-core/ni-parent/pom.xml and the same is present in each project.

cc: @ukgaurav  @sarvkuma